src/containers/Tenant/Diagnostics/Describe/Describe.tsx

```diff
@@ -41,14 +44,29 @@ const Describe = ({path, database, type}: IDescribeProps) => {
     } else if (mergedChildrenPaths) {
         paths = [path, ...mergedChildrenPaths];
     }
-    const {currentData, isFetching, error} = describeApi.useGetDescribeQuery(
+    const {currentDescribe, currentData, isFetching, error} = overviewApi.useGetOverviewQuery(
         paths.length ? {paths, database} : skipToken,
         {
             pollingInterval: autoRefreshInterval,
+            selectFromResult: (props) => {
+                const {currentData} = props;
+                if (!currentData) {
+                    return {currentDescribe: emptyObject, ...props};
+                }
+
+                const mergedData = Object.values(currentData).flat();
```

Comment:
This looks cryptic, I suggest writing it more clearly
```suggestion
                const mergedData = [currentData.data, ...currentData.additionalData];
```